### PR TITLE
PLANET 4399: CPP - Investigate primary and secondary button usage across blocks

### PR DESCRIPTION
**Pull request by pablocubico**
This PR updates all buttons in all blocks in all campaign themes according to the rules documented in these tickets:

https://jira.greenpeace.org/browse/PLANET-4399
https://jira.greenpeace.org/browse/PLANET-4400

As I was having trouble finding the right styles, I renamed some classes and files that were using deprecated block names so it's closer to reality now. Because... "refactoring is not an activity you set aside time to do, refactoring is something you do all the time in little bursts", at least according to Martin Fowler...

### CODE REVIEW

**Comment by galousis** [comment]
Just for mentioning Martin Fowler i will go and review right now your PR

**Comment by galousis** [comment]
@pablocubico in CPP Plastic & Oceans templates its just that the box still has curved corners.

**Comment by pablocubico** [comment]
@galousis Right, I haven't changed any borders or radiuses, just colors!

File: templates/blocks/columns.twig [hunk 1]
```diff
@@ -34,6 +34,7 @@
 									</div>
 								{% endif %}
 
+<<<<<<< HEAD
```
Comment (sagarsdeshmukh) [inline templates/blocks/columns.twig]: Hi @pablocubico - some merge conflicts are git committed. could you please check once?

Comment (pablocubico) [reply templates/blocks/columns.twig]: Ugh! Probably a rebase mistake. Checking...

**Comment by pablocubico** [comment]
@sagarsdeshmukh cleaned and rebased

**Comment by sagarsdeshmukh** [comment]
Tested locally with backstop js visual diff test, looks good to me with my limited experience of css ;)

**Pull request merged**